prow/plugins/label/label.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package label
+
+import (
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/plugins"
+
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+const pluginName = "label"
+
+var (
+	allowedPrefixes = []string{"area", "priority"}
+	regexBuilder    = `(?m)^/%s[\t\s]+(?:\S*(?:[^\n]+))*`
```

Comment:
If this regex does what you want, it's not very clear why.

`[\t\s]+`: you need at least a space after the command, what about `/area$` ?

Can you explain this part: `(?:\S*(?:[^\n]+))*`

0 or more non-space, I assume this is the label, it can be empty?
Followed by non-`\n` characters, and this must be non-empty? What is this supposed to match?